src/App.tsx

```diff
@@ -52,26 +55,42 @@ const RenderDeviceContainers = () => {
         return (
             <div className="pg-mobile-app">
                 <MobileHeader />
-                <AlertsContainer/>
-                <LayoutContainer/>
+                <AlertsContainer />
+                <LayoutContainer />
                 <MobileFooter />
             </div>
         );
     }
 
     return (
         <React.Fragment>
-            <HeaderContainer/>
-            <SidebarContainer/>
-            <CustomizationContainer/>
-            <AlertsContainer/>
-            <LayoutContainer/>
-            <FooterContainer/>
+            <HeaderContainer />
+            <SidebarContainer />
+            <CustomizationContainer />
+            <AlertsContainer />
+            <LayoutContainer />
+            <FooterContainer />
         </React.Fragment>
     );
 };
 
 export const App = () => {
+    // const lang = useSelector(selectCurrentLanguage);
+    // const isMobileDevice = useSelector(selectMobileDeviceState);
+    // return (
+    //     <IntlProvider
+    //         {...{ locale: lang }}
+    //         defaultLocale={lang}
+    //         messages={getTranslations(lang, isMobileDevice)}
+    //         key={lang}>
+    //         <Router history={browserHistory}>
+    //             <Switch>
+    //                 <Route exact={true} path="/" component={SignInScreen} />
+    //             </Switch>
+    //         </Router>
+    //     </IntlProvider>
+    // );
```

Comment:
Please, put here code from "TestComponentWrapper"

Reply:
this is test code, I removed it. This `App` component contains `IntlProvider` and `ReduxProvider` wraps it on upper level